### DISCUSSION TITLE
Pages directory not being detected?

### DISCUSSION THREAD

**Discussion by JeremyQtweb** [nuxt/nuxt#16156] [Questions] [answered]
Hey folks,

I'm experimenting with our first Nuxt3/Vue3/TS app, and coming from Nuxt2 I thought page routing would be the simplest thing.
Right out of the gate it ain't working and I don't know why. It seems like Nuxt3 is not detecting my pages directory and thus not using vue-router. Or... I've done something wrong, so here's my config;

I don't have a `app.vue` file, I am running with a `pages/index.vue` setup;

![image](https://user-images.githubusercontent.com/39040329/206927933-1df62f63-6acb-4780-a7f3-e791a5f121a6.png)

In my `default.vue` layout file I have the following;

```javascript
<template>
  // Vuetify components
  <v-app>
    <v-app-bar />
    <v-navigation-drawer />
    <v-main>
      <slot />
    </v-main>
</template>
```

All the pages simply have the name of the component in an H1 tag i.e. index.vue, about.vue, /experiments/index.vue e.g.

```javascript
<template>
  <div>
    <h1>About.vue</h1>
  </div>
</template>
```

If I run up the app as `localhost:3000/` I see my root index page as expected, with the default template and `<h1>index.vue</h1>` rendered on page.

If I go directly to `localhost:3000/index` I get a 404.

If I go to ANY OTHER page directly, or subfolder, I get a 404 e.g. `localhost:3000/about`, `localhost:3000/experiments/`, `localhost:3000/experiments/charts`.

I haven't bothered trying a `<nuxt-link />` because I know the above should work.

Have I configured something wrong or has Nuxt simply not detected my pages directory?

Cheers for the assist :)

**Comment by TheAlexLichter** [comment]
Hey! Could you kindly provide a [reproduction](https://v3.nuxtjs.org/community/reporting-bugs/#create-a-minimal-reproduction)? 🙏

**Reply by JeremyQtweb** [reply]
Thanks @manniL - I'll put something together ASAP 👍

**Comment by JeremyQtweb** [comment, marked as answer]
Nvm @manniL I managed to figure it out. I must have been expecting Nuxt 3 to be SO amazing that it would just kick the router in without having to reinstall the project AFTER I added the Page directory and pages. Whooops!

Been a while since I set up a new Nuxt project, completely forgot I'd need to do that /doh
